django/contrib/admin/exceptions.py

```diff
@@ -9,3 +9,8 @@ class DisallowedModelAdminLookup(SuspiciousOperation):
 class DisallowedModelAdminToField(SuspiciousOperation):
     """Invalid to_field was passed to admin view via URL query string"""
     pass
+
+
+class FieldIsAForeignKeyColumnName(Exception):
```

Comment:
I guess this is okay. Since the exception is just used as a sort of helper in one file, I'd put the exception there.